components/admin_console/license_settings/renew_license_link_card/renew_license_link_card.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import {FormattedMessage} from 'react-intl';
+
+import moment from 'moment';
+
+import RenewalLink from 'components/announcement_bar/renewal_link/';
+
+import alertIcon from 'images/icons/round-blue-info-icon.svg';
+import warningIcon from 'images/icons/warning-red-icon.svg';
```

Comment:
I believe we may already have these icons in the [compass icons font](https://github.com/mattermost/mattermost-webapp/blob/master/sass/base/_compass-icons.scss). 

I believe it's `alert-circle-outline` for the alert and ` alert-outline` for the warning. You can see an example on how to use them [here](https://github.com/mattermost/mattermost-webapp/blob/master/components/alert_banner.tsx#L27)